ngBoardFrontend/src/app/note/services/note.service.spec.ts

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
Sta mislis da u samom testu postavis listu note-ova sa kojima ce da radi servis, bez obzira na to sto trenutno sam servis ne radi sa serverom? Mislim da onda neces morati da menjas test bez obzira na koju stranu sam servis ode.